configuration/file.md

```diff
@@ -24,7 +24,7 @@ The _Service_ section defines global properties of the service, the keys availab
 | Flush | Set the flush time in seconds. Everytime it timeouts, the engine will flush the records to the output plugin. | 5 |
 | Daemon | Boolean value to set if Fluent Bit should run as a Daemon \(background\) or not. Allowed values are: yes, no, on and off. | Off |
 | Log\_File | Absolute path for an optional log file. |  |
-| Log\_Level | Set the logging verbosity level. Allowed values are: error, info, debug and trace. Values are accumulative, e.g: if 'debug' is set, it will include error, info and debug. Note that _trace_ mode is only available if Fluent Bit was built with the _WITH\_TRACE_ option enabled. | info |
+| Log\_Level | Set the logging verbosity level. Allowed values are: error, info, debug and trace. Values are accumulative, e.g: if 'debug' is set, it will include error, info and debug. Note that _trace_ mode is only available if Fluent Bit was built with the _FLB\_TRACE_ option enabled (previously named _WITH\_TRACE_ ). | info |
```

Comment:
I'd suggest leaving out the mention of `WITH_TRACE`.
Leave the past in the past...

Reply:
Sure thing. Have now removed.